api/rest/workshop_lectors.py

```diff
@@ -1,11 +1,14 @@
 from rest_framework import serializers, viewsets
 
-from .lectors import LectorSerializer
 from ..models import WorkshopLector
 
 
 class WorkshopLectorSerializer(serializers.ModelSerializer):
-    lector = LectorSerializer()
```

Comment:
@PetrDlouhy, that's a good point. Maybe late night programming is not such a good idea after all. Point was to get easier to the lectors name, but this will only complicate things. I will redo this, so Lectors and Roles are loaded in the frontend application as a separate resource and put together on selector level.

Thanks.